docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -524,6 +530,34 @@ context 'when the display name is not present' do
 end
 ----
 
+==== `Suffixes` configuration
+
+[source,ruby]
+----
+# .rubocop.yml
+# RSpec/ContextWording:
+#   Suffixes:
+#     - success
+----
+
+[source,ruby]
+----
+# bad
+context 'when this test is failed' do
+  # ...
+end
+
+# good
+context 'when this test is success' do
```

Comment:
I'd really prefer this to be in an RTL language, as with LTR it's hard to imagine anyone willing to constrain descriptions to be suffixed with a limited set of words.

Reply:
I agree, an RTL language example would work much better here.

Reply:
If it is an RTL language, it appears that if /olloeh$/ is specified, it will correctly check if it ends in olloeh as an RTL language.

```ruby
# bad
context '!sknaht olleh'
end

# good
context 'olleh !sknaht'
end
```

How about using Japanese as an example, as in the case of the original Issue?